src/state/persisted/schema.ts

```diff
@@ -54,6 +54,7 @@ export const schema = z.object({
   }),
   hiddenPosts: z.array(z.string()).optional(), // should move to server
   useInAppBrowser: z.boolean().optional(),
+  dataSaverEnabled: z.boolean(),
```

Comment:
Needs to be optional or else pre-existing data will fail validation and be discarded